.github/workflows/release_linux.yml

```diff
@@ -52,9 +52,16 @@ jobs:
       run: |
         cp build/dist/Outline-Client.AppImage outline-releases/client/stable/Outline-Client.AppImage
         cp build/dist/linux-latest.yml outline-releases/client/stable/linux-latest.yml
-        RELEASE_VERSION="$(node scripts/get_version.mjs linux)""
+        RELEASE_VERSION="$(node scripts/get_version.mjs linux)"
         cd outline-releases
         git add client/stable/Outline-Client.AppImage client/stable/linux-latest.yml
         git commit -m "Release linux client ${RELEASE_VERSION}"
         git tag "linux-${RELEASE_VERSION}"
-        git push origin master
\ No newline at end of file
+        git push origin master
+    
+    - name: Publish to Snap store    
```

Comment:
Is this working? Did you create a team account on the Snap store?
Let's move this to a separate PR.

Reply:
Ah, sorry, I just commented this out for later reference. Just re-requested ya.